common/goal_distance_calculator/launch/goal_distance_calculator.launch.xml

```diff
@@ -1,8 +1,6 @@
 <launch>
-  <arg name="oneshot" default="false"/>
```

Comment:
@kaspermeck-arm
I mean one shot parameter is set false by launcher

Reply:
@taikitanaka3 
Aha, I see. Thanks for pointing that out!